platform/darwin/src/MGLStyle.h

```diff
@@ -610,6 +610,16 @@ MGL_EXPORT
  */
 @property (nonatomic, strong) MGLLight *light;
 
+#pragma mark Managing Style's localization
+
+/**
+ Set to true to determine the localization according to the languages
+ supported by <a href="https://www.mapbox.com/vector-tiles/mapbox-streets-v7/#overview">Mapbox Streets</a>, 
+ it will look for the Preferred Language Order setting in the Language & Region section of Settings. 
+ If it doesn't fing a match it will fall back to english localization.
+ */
+@property (nonatomic, getter=isLocalizingLabels) BOOL localizingLabels;
```

Comment:
The correct tense here would be `localizesLabels`; there’s no need for a custom getter name in that case.